### DISCUSSION TITLE
How to sign out of a github session?

### DISCUSSION THREAD

**Discussion by jshgwave** [community/community#23023] [New to GitHub]
<p>How does a person sign out of a github session?  None of the books and tutorials seem to discuss this.</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/jshgwave">@jshgwave</a>,</p>
<p>If you click on your avatar in the top right corner, a menu will pop up. This menu has a sign out option :slight_smile:</p>

**Reply by ADogNamedBoo** [reply]
Stupid question: Is "sign out" the same as "log out"? I was wondering if sign out means finishing my github account.

**Comment by jshgwave** [comment]
<p>Thanks!</p>
